### PR TITLE
[1.12] Hotfix: Fix placement tables

### PR DESCRIPTION
**Pull request by cicoyle**
# Description

cherry-picked  https://github.com/dapr/dapr/pull/7197 into release-1.12

release notes: https://github.com/dapr/dapr/pull/7245

### CODE REVIEW

**Comment by cicoyle** [comment]
oops, I guess there is a missing ref from the cherry-pick. Looking...

**Comment by JoshVanL** [comment]
@cicoyle please include [release notes](https://github.com/dapr/dapr/tree/master/docs/release_notes) of the change.

**Comment by cicoyle** [comment]
I'll add this in and fix the issues after my current meeting

**Review by mukundansundar — DISMISSED** [review]

File: pkg/placement/membership.go [hunk 1]
```diff
@@ -359,77 +356,88 @@ func (p *Service) performTablesUpdate(ctx context.Context, hosts []placementGRPC
 	// Otherwise, each Dapr runtime will have inconsistent hashing table.
 	startedAt := p.clock.Now()
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
-	defer cancel()
+	// Enforce maximum API level
```
[the remark this answers is not in the document]
Comment (JoshVanL) [reply pkg/placement/membership.go:359]: We should remove all “level” related code from the patch as this was (will be) introduced in 1.13.

**Comment by cicoyle** [comment]
Are the macOS unit tests for `TestProxySuite/TestPingStream_MultipleThreads` && `TestProxySuite/TestRecoveryFromNetworkFailure` typically flaky?

**Review by ItalyPaleAle — APPROVED** [review]

**Comment by JoshVanL** [comment]
@cicoyle @artursouza PR is missing release notes.

**Pull request merged**